maintenance/aws-janitor/Makefile

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 IMAGE ?= gcr.io/k8s-test-infra-aws/aws-janitor
-VERSION ?= 0.4
+VERSION = $(shell date +v%Y%m%d)-$(shell git describe --tags --always --dirty)
```

Comment:
I'd probably stick with `?=` so we can specify a static version, but in practice we don't want to IMO